src/components/SideEvent/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactMarkdown from 'react-markdown';
+
+import { Card } from '@material-ui/core';
+
+import {
+  Wrapper,
+  CardContainer,
+  Description,
+  DateGroup,
+  DateIconStyled,
+  Date,
+  Underline,
+  Title,
+  LocationItems,
+  LocationIconStyled,
+  Location,
+  DescriptionContainer,
+  ButtonsGroup,
+  Link,
+  ButtonStyle,
+  ActionPackButton,
+} from './styled';
+
+const callDescription = description => (
+  <DescriptionContainer component="div">
+    <ReactMarkdown source={description} />
+  </DescriptionContainer>
+);
+
+const callLearnMoreButton = (slug, learnMore) => {
+  if (!learnMore) {
+    return null;
```

Comment:
Another way to do this would be something like 
```jsx
learnMore ? callLearnMoreButton(slug)
```
where you call callLearnMoreButton https://github.com/vulekamali/vulekamali-create/pull/33/files#diff-8b15179eb53e60f15b2274e3b65b8b03R87